terraform/aks/databases.tf

```diff
@@ -28,10 +28,9 @@ module "redis" {
   service_name          = local.service_name
   service_short         = var.service_short
   config_short          = var.config_short
-
   cluster_configuration_map = module.cluster_data.configuration_map
-
   use_azure               = var.deploy_azure_backing_services
   azure_enable_monitoring = var.enable_monitoring
   azure_patch_schedule    = [{ "day_of_week" : "Sunday", "start_hour_utc" : 01 }]
+  azure_maxmemory_policy    = "noeviction"
```

Comment:
This would probably rebuild the whole redis server. Please discuss it with the devs.

Reply:
removed as agreed